nginx.conf

```diff
@@ -298,6 +298,10 @@ server {
   rewrite ^/platform/actors/development/testing-and-maintenance$ /platform/actors/development/deployment/automated-tests redirect;
   rewrite ^/platform/actors/development/source-code$             /platform/actors/development/deployment/source-types redirect;
 
+  # Rename output schema to dataset schema
+  rewrite ^/platform/actors/development/actors-definition/output-schema$ /platform/actors/development/actors-definition/dataset-schema permanent;
+  rewrite ^academy/deploying-your-code/ouptup-schema$ /academy/deploying-your-code/dataset-schema permanen;
```

Comment:
Typo here: "ouptup" and "permanen"

Reply:
Thank you, fixed, will move forward and merge it